rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/docker/DockerImageNameFactory.groovy

```diff
@@ -13,8 +13,12 @@ class DockerImageNameFactory extends ImageNameFactory {
 
   @Override
   def buildImageName(BakeRequest bakeRequest, List<PackageNameConverter.OsPackageName> osPackageNames) {
+    String imageName = bakeRequest.extended_attributes?.get('docker_target_image_name')
```

Comment:
Yeah, I had a vague memory of us discussing this a while back. I seemed to remember that we wanted to avoid using the ami_name for other providers than AWS but I might be wrong. 

Anyway, I have no strong feelings on this so I don't mind using the ami_name if that's something that we want. Keep ami_name?